src/Directory.Build.props

```diff
@@ -2,6 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <Import Project="$(MSBuildThisFileDirectory)../build/IceRpc.Src.props" />
     <PropertyGroup>
+        <!-- The target framework to build assemblies in src directory, the default value $(IceRpcTargetFramework) is
+             set in ./build/IceRpc.Version.props
+             -->
+        <TargetFramework>$(IceRpcTargetFramework)</TargetFramework>
```

Comment:
We build all projects under `src` with this target framework.

Reply:
Except for the template projects that set `<TargetFramework>net8.0</TargetFramework>` and therefore overwrite this setting?

It's not clear to me how we handle the templates.

Reply:
We don't handle templates yes, as templates are just packaged but not built. We need to add a template parameter if we want to support multiple frameworks with the templates.

Reply:
Added a Framework parameter that allows you to choose net8.0|net9.0 target frameworks.